kategory/src/test/kotlin/kategory/data/MonoidKTests.kt

```diff
@@ -0,0 +1,26 @@
+package kategory
+
+import io.kotlintest.KTestJUnitRunner
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class MonoidKTests : UnitSpec() {
```

Comment:
This has to be removed. Forgot about it. MonoidK and SemigroupK laws are tested inside all their implementations test classes (I'm already doing that). These are basically repeated tests not adding value.